source/reference/config/triggers.txt

```diff
@@ -96,15 +118,26 @@ Database Triggers
 
 Database trigger configurations conform to the base trigger schema with
 additional configuration options that specify which collection to watch and when
-to fire the trigger. The following fields exist in *database* and *deployment* 
-trigger configuration files:
+to fire the trigger. 
+
+Below are two configurations: one configuration is 
+for a trigger with a Function event processor and the other is for a trigger with 
+an AWS EventBridge event processor. The only difference between the two configuraitons
+is the type of ``event_processor`` and the presence of an ``error_handler`` in the trigger
+with AWS EventBridge. The following fields exist in *database* and *deployment* 
+trigger configuration files. 
+
+.. _trigger-function-event-processor:
+
+**Database Trigger with a Function Event Processor**
 
 .. code-block:: json
    :caption: triggers/<trigger name>.json
    
    {
      "name": "<Trigger Name>",
      "type": "DATABASE",
+     "disabled": <boolean>,
      "config": {
```

Comment:
compared to database trigger configs i'm generating, this looks like it's missing `"skip_catchup_events": <boolean>,` and `"maximum_throughput": <boolean>,`

Reply:
would need to add them to the example and the defintions